src/main/java/com/stripe/param/TokenCreateParams.java

```diff
@@ -0,0 +1,496 @@
+  @SerializedName("bank_account")
+  BankAccount bankAccount;
+
+  @SerializedName("card")
+  Object card;
+
+  @SerializedName("currency")
+  String currency;
+
+  /**
+   * The customer (owned by the application's account) for which to create a token. For use only
+   * with [Stripe Connect](https://stripe.com/docs/connect). Also, this can be used only with an
+   * [OAuth access token](https://stripe.com/docs/connect/standard-accounts) or [Stripe-Account
+   * header](https://stripe.com/docs/connect/authentication). For more details, see [Shared
+   * Customers](https://stripe.com/docs/connect/shared-customers).
+   */
+  @SerializedName("customer")
+  String customer;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /** The PII this token will represent. */
+  @SerializedName("pii")
+  Pii pii;
+
+  private TokenCreateParams(
+      BankAccount bankAccount,
+      Object card,
+      String currency,
+      String customer,
+      List<String> expand,
+      Pii pii) {
+    this.bankAccount = bankAccount;
+    this.card = card;
+    this.currency = currency;
+    this.customer = customer;
+    this.expand = expand;
+    this.pii = pii;
+  }
+
+  public static Builder builder() {
+    return new com.stripe.param.TokenCreateParams.Builder();
+  }
+
+  public static class Builder {
+    private BankAccount bankAccount;
+
+    private Object card;
+
+    private String currency;
+
+    private String customer;
+
+    private List<String> expand;
+
+    private Pii pii;
+
+    /** Finalize and obtain parameter instance from this builder. */
+    public TokenCreateParams build() {
+      return new TokenCreateParams(
+          this.bankAccount, this.card, this.currency, this.customer, this.expand, this.pii);
+    }
+
+    /**
+     * Add all elements to `expand` list. A list is initialized for the first `add/addAll` call, and
+     * subsequent calls adds additional elements to the original list. See {@link
+     * TokenCreateParams#expand} for the field documentation.
+     */
+    public Builder addAllExpand(List<String> elements) {
+      if (this.expand == null) {
+        this.expand = new ArrayList<>();
+      }
+      this.expand.addAll(elements);
+      return this;
+    }
+
+    /**
+     * Add an element to `expand` list. A list is initialized for the first `add/addAll` call, and
+     * subsequent calls adds additional elements to the original list. See {@link
+     * TokenCreateParams#expand} for the field documentation.
+     */
+    public Builder addExpand(String element) {
+      if (this.expand == null) {
+        this.expand = new ArrayList<>();
+      }
+      this.expand.add(element);
+      return this;
+    }
+
+    /** The bank account this token will represent. */
+    public Builder setBankAccount(BankAccount bankAccount) {
+      this.bankAccount = bankAccount;
+      return this;
+    }
+
+    public Builder setCard(Card card) {
```

Comment:
Removed ApplyPay, SwipeData etc..
Name was changed from `CreditCard` to `Card` in this PR